src/clims/plugins/demo/dnaseq/handlers/create_example_data.py

```diff
@@ -77,7 +77,7 @@ def handle(self):
         # Create workbatch to be used at wip workbatch details page. The
         # name of the workbatch has to be known here.
         workbatch = ExampleWorkBatch(name='wip-workbatch', status=1)
-        workbatch.kit_type = "X"
+        # workbatch.kit_type = "X"
```

Comment:
Should this be here? (If so please comment why)

Reply:
I have now pushed another commit to fix this.